app/routes.py

```diff
@@ -1 +1,164 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task 
+from app.models.goal import Goal 
+from flask import Blueprint, make_response, abort, request, jsonify
+from datetime import datetime
+from dotenv import load_dotenv
+load_dotenv()
+from app.helpers import *
+
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+
+#************Task Routes*****************
+
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    return create_new_model(Task, request.get_json())
+
+
+@task_bp.route("", methods=["GET"])
+def get_all():
+    task_query = Task.query
+    title_query = request.args.get("sort")
```

Comment:
Consider renaming `title_query` to a name that captures that sorting will happen based on its value. Maybe like `title_sort_query` or `sort_query`